swap/peer.go

```diff
@@ -40,6 +40,7 @@ type Peer struct {
 	contractAddress    common.Address
 	lastReceivedCheque *Cheque
 	lastSentCheque     *Cheque
+	pendingCheque      *Cheque
```

Comment:
A small comment to each field in the peer struct would be nice.

Reply:
done